packages/ymax-planner/src/engine.ts

```diff
@@ -352,6 +347,64 @@ export const startEngine = async ({
     portfolioKeyForDepositAddr.set(depositAddress, portfolioKey);
   }).done;
 
+  console.warn('Initializing subscription tracking...');
+  try {
+    const subscriptionKeys = await vstorageKit.vstorage.keys(
+      ORCHESTRATION_SUBSCRIPTIONS_PREFIX,
+    );
+    console.warn(
+      `Found ${subscriptionKeys.length} existing subscriptions to monitor`,
+    );
+
+    // Process existing pending subscriptions on startup
+    await makeWorkPool(subscriptionKeys, undefined, async subscriptionKey => {
+      try {
+        const subscriptionData = (await vstorageKit.readPublished(
+          `${stripPrefix('published.', ORCHESTRATION_SUBSCRIPTIONS_PREFIX)}.${subscriptionKey}`,
```

Comment:
Probably more readable (here and above with `VSTORAGE_PATH_PREFIX`):
```suggestion
          stripPrefix('published.', `${ORCHESTRATION_SUBSCRIPTIONS_PREFIX}.${subscriptionKey}`)
```